.ci/scripts/conformance-scripts/download_tests.sh

```diff
@@ -21,31 +21,37 @@ SKLEARN_PATH="$CONDA_PREFIX/lib/python${PYTHON_VERSION}/site-packages/sklearn/"
 
 case ${ALG_NAME} in
     "dbscan")
-        cp ${SKLEARN_PATH}cluster/tests/test_dbscan.py test_dbscan.py
+        wget -O test_dbscan.py ${SKLEARN_URL_ROOT}cluster/tests/test_dbscan.py
     ;;
     "elastic_net")
-        cp ${SKLEARN_PATH}linear_model/tests/test_coordinate_descent.py test_elastic_net.py
+        wget -O test_elastic_net.py ${SKLEARN_URL_ROOT}linear_model/tests/test_coordinate_descent.py
     ;;
     "kmeans")
-        cp ${SKLEARN_PATH}cluster/tests/test_k_means.py test_kmeans.py
+        wget -O test_kmeans.py ${SKLEARN_URL_ROOT}cluster/tests/test_k_means.py
     ;;
     "lin_reg")
-        cp ${SKLEARN_PATH}linear_model/tests/test_base.py test_lin_reg.py
+        wget -O test_lin_reg.py ${SKLEARN_URL_ROOT}linear_model/tests/test_base.py
     ;;
     "log_reg")
-        cp ${SKLEARN_PATH}linear_model/tests/test_logistic.py test_log_reg.py
+        wget -O test_log_reg.py ${SKLEARN_URL_ROOT}linear_model/tests/test_logistic.py
     ;;
     "pca")
-        cp ${SKLEARN_PATH}decomposition/tests/test_pca.py test_pca.py
+        wget -O test_pca.py ${SKLEARN_URL_ROOT}decomposition/tests/test_pca.py
     ;;
     "ridge_reg")
-        cp ${SKLEARN_PATH}linear_model/tests/test_ridge.py test_ridge_reg.py
+        wget -O test_ridge_reg.py ${SKLEARN_URL_ROOT}linear_model/tests/test_ridge.py
     ;;
     "svm")
-        cp ${SKLEARN_PATH}svm/tests/test_svm.py test_svm.py
+        wget -O test_svm.py ${SKLEARN_URL_ROOT}svm/tests/test_svm.py
     ;;
     "svm_sparse")
-        cp ${SKLEARN_PATH}svm/tests/test_sparse.py test_svm_sparse.py
+        wget -O test_svm_sparse.py ${SKLEARN_URL_ROOT}svm/tests/test_sparse.py
+    ;;
+    "forest")
+        wget -O test_forest.py ${SKLEARN_URL_ROOT}ensemble/tests/test_forest.py
+    ;;
+    "knn")
+        wget -O test_knn.py ${SKLEARN_URL_ROOT}neighbors/tests/test_neighbors.py
```

Comment:
In this file (download_tests.sh), need only to add copy test files for forest and knn.
```suggestion
    "forest")
        cp ${SKLEARN_PATH}ensemble/tests/test_forest.py test_forest.py
    ;;
    "knn")
        cp ${SKLEARN_PATH}neighbors/tests/test_neighbors.py test_knn.py
```

Reply:
For other algorithms leave it as it was before the changes

Reply:
Okay, corrected